<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceManager</class>
 <widget class="QDialog" name="PreferenceManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Indstillinger</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/logo.ico</normaloff>:/img/logo.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labLanguage">
         <property name="text">
          <string>Sprog:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labTimer">
         <property name="text">
          <string>Tidtager:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="cbLanguage">
         <item>
          <property name="text">
           <string notr="true">Dansk</string>
          </property>
          <property name="icon">
           <iconset resource="img.qrc">
            <normaloff>:/img/Flag_DK.png</normaloff>:/img/Flag_DK.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Føroyskt</string>
          </property>
          <property name="icon">
           <iconset resource="img.qrc">
            <normaloff>:/img/Flag_FO.png</normaloff>:/img/Flag_FO.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">English</string>
          </property>
          <property name="icon">
           <iconset resource="img.qrc">
            <normaloff>:/img/Flag_UK.png</normaloff>:/img/Flag_UK.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbTimer">
         <item>
          <property name="text">
           <string>Skjul tidtager</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vis tid, når jeg er færdig</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vis tidtager</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Færdig</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
